boa/src/builtins/object/tests.rs

```diff
@@ -289,3 +289,83 @@ fn object_is_prototype_of() {
 
     assert_eq!(context.eval(init).unwrap(), JsValue::new(true));
 }
+
+#[test]
+fn object_get_own_property_names_invalid_args() {
+    let error_message = r#"Uncaught "TypeError": "cannot convert 'null' or 'undefined' to object""#;
+
+    check_output(&[
+        TestAction::TestEq("Object.getOwnPropertyNames()", error_message),
+        TestAction::TestEq("Object.getOwnPropertyNames(null)", error_message),
+        TestAction::TestEq("Object.getOwnPropertyNames(undefined)", error_message),
+    ]);
+}
+
+#[test]
+fn object_get_own_property_names() {
+    let scenario = r#"
+        const a = Object.getOwnPropertyNames(0);
+        const b = Object.getOwnPropertyNames(false);
+        const c = Object.getOwnPropertyNames(Symbol("a"));
+        const d = Object.getOwnPropertyNames({});
+        const e = Object.getOwnPropertyNames(NaN);
+        const f = Object.getOwnPropertyNames([1, 2, 3]);
+        const g = Object.getOwnPropertyNames({
+            "a": 1,
+            "b": 2,
+            [ Symbol("c") ]: 3,
+            [ Symbol("d") ]: 4,
+        });
```

Comment:
I think it's better when the test statement is together with the result of the call, like in `object_get_own_property_symbols_invalid_args`. That way you don't need to look up and down to see which one is the expected result of the call. What do you think?

Reply:
Yeah that makes sense! Let me make the change.